### DISCUSSION TITLE
Variables Support

### DISCUSSION THREAD

**Discussion by zeromdg** [HadesArchitect/GrafanaCassandraDatasource#121] [General]
Hi Hades Team!

I'm very thankfull for this development, now I can use the data that I have of a cassandra cluster.

My question is related to the use of Variables for a dashboard. How can I define variables using this datasource?

![image](https://user-images.githubusercontent.com/5429299/147756821-450d8e32-864d-48de-8014-0989cdad364f.png)

Thank you very much and happy holidays

**Comment by HadesArchitect** [comment]
Hi @zeromdg, thank you for the warm feedback! That is an amazing feature you mention, sadly it's not yet supported by the data source - we have ticket #100 but it isn't scheduled yet. =(

**Reply by HadesArchitect** [reply]
We accept pull requests if you are into typescript :)

**Comment by futuarmo** [comment]
Hello, @zeromdg! Please, describe some use cases of variables with our datasource to evaluate posibility to add it. It will be great, if you will describe it in details. Thanks

**Comment by HadesArchitect** [comment]
Closed in #177
